core/api/service/system/requests/health.cpp

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "api/service/system/requests/health.hpp"
+#include "api/jrpc/value_converter.hpp"
+
+namespace kagome::api::system::request {
+
+  Health::Health(std::shared_ptr<SystemApi> api) : api_(std::move(api)) {
+    BOOST_ASSERT(api_ != nullptr);
+  }
+
+  outcome::result<void> Health::init(
+      const jsonrpc::Request::Parameters &params) {
+    if (!params.empty()) {
+      throw jsonrpc::InvalidParametersFault("Method should not have params");
+    }
+    return outcome::success();
+  }
```

Comment:
We can use RequestBase to avoid multiple params initialization of the same type in each class.

Reply:
I suggest to recactor it after feature-owner of RequestBase will provide some examples of using. ;)